weaver/datatype.py

```diff
@@ -605,6 +605,20 @@ def process(self, process):
             raise TypeError("Type 'str' is required for '{}.process'".format(type(self)))
         self["process"] = process
 
+    @property
+    def type(self):
+        # type: () -> str
+        """
+        Obtain the type of the element associated to the creation of this job.
+
+        .. seealso::
+            - Defined in |ogc-statusInfo|
+            - Queried with https://docs.ogc.org/DRAFTS/18-062.html#_parameter_type
+        """
+        if self.service is None:
+            return "process"
```

Comment:
I don't know how the type is used, but it looks like a variable could be helpful to compare to.